packages/shared/src/components/cards/v1/ActionButtons.tsx

```diff
@@ -152,23 +152,17 @@ export default function ActionButtons({
           <Link href={post.commentsPermalink}>
             <Button
               id={`post-${post.id}-comment-btn`}
-              className="pointer-events-auto ml-2"
+              className="pointer-events-auto ml-2 hover:text-theme-color-blueCheese"
               color={ButtonColor.BlueCheese}
-              icon={<CommentIcon secondary={post.commented} />}
```

Comment:
I think this was intended right? When the user has made a comment we fill it.

Reply:
Ah yes, could be, it's not in designs, but if this is what the behaviour used to be, should be the same. 
Would `post.commented` check if the user itself has commented? Or if the post is commented by any user?

Reply:
Yeah, I checked the Figma and it doesn't seem to be there. It checks whether the logged in user has made a comment.

Reply:
Have updated to only make the required change. thank u!!